upgrade/src/upgradeBridgeOnForeign.js

```diff
@@ -0,0 +1,51 @@
+require('dotenv').config()
+const Web3 = require('web3')
+const { newImplementation } = require('./constants')
+const multiSigWalletAbi = require('../abi/multiSigwallet')
+const proxyAbi = require('../../build/contracts/EternalStorageProxy').abi
+const confirmTransaction = require('./confirmTransaction')
+
+const {
+  FOREIGN_PRIVKEY,
+  FOREIGN_RPC_URL,
+  FOREING_BRIDGE_ADDRESS,
+  ROLE,
+  FOREIGN_START_BLOCK,
+  FOREIGN_GAS_PRICE
+} = process.env
+
+const web3 = new Web3(new Web3.providers.HttpProvider(FOREIGN_RPC_URL))
+const { address } = web3.eth.accounts.wallet.add(FOREIGN_PRIVKEY)
+
+const upgradeBridgeOnForeign = async () => {
+  try {
+    const proxy = new web3.eth.Contract(proxyAbi, FOREING_BRIDGE_ADDRESS)
+    const ownerAddress = await proxy.methods.upgradeabilityOwner().call()
+    const multiSigWallet = new web3.eth.Contract(multiSigWalletAbi, ownerAddress)
+    // 0x46016a67 = upgradeToV250()
```

Comment:
you use `bridge.methods.migrateToMCD().encodeABI()` in `migrateToMCD.js`. Do you think it is worth to use similar approach here?

Reply:
Yes, it makes sense to use a similar approach. Updated https://github.com/poanetwork/tokenbridge-contracts/pull/337/commits/79d21103b6f716e23da74c972a09e8d0b29db256